### PR TITLE
Per Issue 83, dif btw create, open, import proj

### PR DESCRIPTION
**Pull request by jas58**
Added @bencomp 's ([suggested text](https://github.com/LibraryCarpentry/lc-open-refine/issues/83)) to beginning of "import project"
#83

### CODE REVIEW

**Comment by jas58** [comment]
Should I add from #208 which seems to be about a similar part of the lesson?
It might be an instructor note to really slow down at this intro because the learners are quickly lost between a completely unfamiliar screen (think map of a neighborhood in a foreign language) and hopping between your screen, own screen, Zoom window, notes, etc.

suggestion read: "
Point 9 says the project name box is in the upper right corner, but in fact it is located in the middle of the blue menu bar above the columns of data.

Going Further:
"Look at the other options on the Import screen - try changing some of these options and see how that changes the Preview and how the data appears after import."
Being very new to all this I could not see how to try some of the other options on the Import screen, as by the time you read this sentence you have already clicked 'Create Project' and the Import screen/preview window has gone. It took me awhile to figure out that I would need to click on the diamond icon in the top left corner, to start over and create a new project in order to experiment with some of the other options on the Import screen."

**Comment by bencomp** [comment]
Thanks for opening this, @jas58! I forgot to put it on my to-do list.
It looks like the discussion on #208 has not reached a suggestion for wording, so that may become a separate PR.

File: _episodes/02-importing-data.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ keypoints:
 
 ## Importing data
 
+OpenRefine does not manipulate your data directly. Instead, the data you import and all the changes you make are stored in a project. You can stop working on a project and continue later if you like.
+When you want to 'refine' a new file, you start by creating a new project. When you want to continue working on a project, you can open it through "Open Project".
+It is also possible to export a project on one computer and continue working on it on a different computer. To do so, you transfer the exported files to the new computer and use "Import Project" on the new computer.
```
Comment (bencomp) [inline _episodes/02-importing-data.md:19]:
```suggestion
OpenRefine does not manipulate your data directly.
Instead, the data you import and all the changes you make are stored in a project.
You can stop working on a project and continue later if you like.
When you want to 'refine' a new file, you start by creating a new project.
When you want to continue working on a project, you can open it through "Open Project".
It is also possible to export a project on one computer and continue working on it on a different
computer.
To do so, you transfer the exported files to the new computer and use "Import Project" on the new
computer.
```

**Review by bencomp — APPROVED** [review]
My approval shouldn't really count, as it was my suggestion. I suggested extra line breaks to stay under the 100 characters on each line. This is not enforced, but I wouldn't want new text to break the rules. :)

**Review by bencomp — APPROVED** [review]

**Review by ostephens — APPROVED** [review]

**Pull request merged**